app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/AccessApiFragment.kt

```diff
@@ -200,27 +221,28 @@ class AccessApiFragment : Fragment() {
             STATUS.SignedIn -> {
                 binding.signIn.isEnabled = false
                 binding.signOut.isEnabled = true
-                binding.getApi.isEnabled = true
+                binding.getApi1.isEnabled = true
+                binding.getApi2.isEnabled = true
             }
             STATUS.SignedOut -> {
                 binding.signIn.isEnabled = true
                 binding.signOut.isEnabled = false
-                binding.getApi.isEnabled = false
+                binding.getApi1.isEnabled = false
+                binding.getApi2.isEnabled = false
             }
         }
     }
 
     private fun emptyResults() {
+        binding.result.text = ""
         binding.resultText.text = ""
     }
 
-    private fun displayAccount(accountState: AccountState) {
+    private fun displayAccount(accountState: AccountState, scopes: List<String> = listOf("openid", "offline_access", "profile") ) {  // The default scopes added by the SDK
```

Comment:
Can you please explain why we need a default value for the scopes?
I thought we wanted to let the external developer specifies the scopes in this case.

Reply:
This `displayAccount` funtion is used when onResume phase of the fragment as well as successfully `signIn()`. The sample app passes no scopes (applies default scopes) when calling `signIn` interface. Thus, explicitly declare the specific sign in scopes in this case to get the corresponding access token.

Reply:
Thanks for explanation Yuki. If this method is called during the fragment onResume method, it means that you always retrieve and show an access token related to default scopes when the sample app goes in background and then back in foreground?
On iOS we retrieve the access token only when the user taps the button "getAPI1" or "getAPI2".

Reply:
That's a good point! Yes, it may caused confusion for the users if they tap the button "getAPI1" or "getAPI2" then the sample app goes in background and back in foreground. However, if the automatic token display is removed, then the login and logout statuses are less visible and it doesn't seem like a good experience. May I ask how IOS side deal with this problem?

Reply:
I think we have the same issue on iOS too 😂. An idea would be:
- at the onCreate we retrieve the cached account and show just the email
- We retrieve the access token only when the user presses the button "getAPI1" or "getAPI2"
- onResume we just don't do anything

I don't know if this can work, but I'd give it a try

Reply:
I think that's a good suggestion @nilo-ms. When the app is launched/brought to foreground, only show account information. Show the access token as part of a dedicated button click.